### ISSUE TITLE
Getting started

### ISSUE THREAD

**Issue by mgifford** [cvs-health/testaro#41]
I wanted to check out this package and started with:

` % npm i testaro
` 
 
 That got me:
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@siteimprove%2falfa-act - Not found
npm ERR! 404 
npm ERR! 404  '@siteimprove/alfa-act@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Not very pretty.

It would be nice if maybe it just ran with the tools where there was a registry.

I don't need to run with the whole suite of tests at first, just those which are easy to get me started. I'd think that axe & Qualweb would be a good starting point.

**Comment by jrpool** [comment]
You are correct that, for a typical user, the usual installation command will fail. That is because Siteimprove has chosen to distribute its Alfa package through GitHub Packages. This is spelled out in detail in the Installation section of the README file (https://github.com/cvs-health/testaro/blob/main/README.md#installation). @mgifford: Do you think that we should include a warning about this earlier in the README, or move the Installation section to an earlier location in the file?
